AzureResources/azuredeploy.parameters.json

```diff
@@ -12,19 +12,19 @@
       "value": "AddName-ai"
     },
     "appInsightsLocation": {
-      "value": "AddGeography"
+      "value": "australiaeast"
```

Comment:
@hmdvs Are we expecting that this will stay hardcoded to australiaeast ? or should we be injecting the value during provisioning.

Reply:
We need to get this from provision. The user needs to select the correct Azure Subscription linked to the user logged into the Azure CLI